File: D108/src/main/java/com/example/model/dto/PictureDto.java
package com.example.model.dto;

import lombok.*;

@Getter
@Setter
@ToString
@NoArgsConstructor
@AllArgsConstructor
public class PictureDto {
    private int pictureId;
    private int userId;
    private String imageUrl;
    private String topic;
    private String title;
    private String description;
    private boolean isDisplayed;
    private int rotation;
    private float xVal;
    private float yVal;
    private int furniture;
}

File: GameTset/app/src/main/java/com/example/gametset/room/data/model/dto/PictureDto.kt
package com.example.gametset.room.data.model.dto

// 데이터 모델
data class PictureDto(
    val pictureId: Int,
    val userId: Int,
    val imageUrl: String,
    val topic: String = "",
    val title: String = "",
    val description: String,
    val rotation: Int,
    val furniture: Int,  // 액자 아이템 ID
    val xval: Float,
    val yval: Float,
    val displayed: Boolean,
)